.claude/commands/gh-fix-lint.md

```
@@ -13,43 +13,64 @@ First, extract the PR number if a URL was provided and get PR details:
-If there are changes, commit and push them:
+If there are changes, commit them. Only stage the modified files (not untracked files that may be local):
 
-RUN git add -A
+RUN git add -u
 RUN git commit -m "chore: fix lint and formatting issues"
-RUN git push origin <branch-name>
 
-## Step 6: Return to Original Branch
+Push to the contributor's fork using the owner from Step 1:
+
+RUN git push https://github.com/<owner-from-step-1>/<repo-name>.git HEAD:<branch-name-from-step-1>
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)